example/android/app/build.gradle

```diff
@@ -93,9 +93,9 @@ android {
 
     compileSdkVersion rootProject.ext.compileSdkVersion
 
-    namespace "com.movableink"
+    namespace "com.movableink.app"
```

Comment:
changed the appId to match the other test apps using the  Digital Asset Links file